.github/workflows/preDeploy.yml

```diff
@@ -44,11 +44,31 @@ jobs:
           GITHUB_TOKEN: ${{ github.token }}
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK }}
 
+  # Get details about the merged pull request
+  getMergedPullRequest:
+    runs-on: ubuntu-latest
+
+    outputs:
+      title: ${{ steps.getMergedPullRequest.outputs.title }}
+      body: ${{ steps.getMergedPullRequest.outputs.body }}
+      number: ${{ steps.getMergedPullRequest.outputs.number }}
+      labels: ${{ steps.getMergedPullRequest.outputs.labels }}
+      assignees: ${{ steps.getMergedPullRequest.outputs.assignees }}
+      author: ${{ steps.getMergedPullRequest.outputs.author }}
+
+    steps:
+      - name: Get merged pull request
+        id: getMergedPullRequest
+        # TODO: Point back action actions-ecosystem after https://github.com/actions-ecosystem/action-get-merged-pull-request/pull/223 is merged
```

Comment:
Ah that is just copy pasta fromwhat was there. I see the PR is still not merged. I can create an issue and remove the TODO. 

@roryabraham since you should be notified once that PR will be merged, I would assign you to that issue

Reply:
Issue is here https://github.com/Expensify/Expensify/issues/201602

Accepting the suggestion